scripts/create_perf_json.py

```diff
@@ -1086,8 +1107,10 @@ def extract_tma_metrics(self, csvfile: TextIO, pmu_prefix: str,
         tma_cpu = None
         if self.shortname == 'BDW-DE':
             tma_cpu = 'BDW'
-        if self.shortname == 'ADLN':
+        elif self.shortname == 'ADLN':
             tma_cpu = 'GRT'
+        elif self.shortname == 'SRF':
```

Comment:
Ian, would you mind also adding Grand Ridge (GRR)? Like SRF, GRR is also Crestmont based.

```python
elif self.shortname in ['SRF', 'GRR']:
```